.gitignore

```diff
@@ -380,3 +380,8 @@ pyrightconfig.json
 
 # End of https://www.toptal.com/developers/gitignore/api/c++,meson,sonar,python,sonarqube,visualstudiocode,clion
 .idea
+
+# Colcon
```

Comment:
Because I was calling `colcon build` directly from the interface lib directory. But I guess the entire lib should be in a parent ros workspace, and I should call `colcon build` from there? I'll remove the changes from the .gitignore